src/info/models.py

```diff
@@ -29,15 +31,33 @@ class Meta:
         default=100,
     )
 
+    team = models.ForeignKey(
+        'teams.Team',
+        null=True,
+        blank=True,
+        help_text='The team responsible for this info category.',
+        on_delete=models.PROTECT,
+        related_name='info_categories'
+    )
+
     def clean(self):
         if InfoItem.objects.filter(category__camp=self.camp, anchor=self.anchor).exists():
             # this anchor is already in use on an item, so it cannot be used (must be unique on the page)
-            raise ValidationError({'anchor': 'Anchor is already in use on an info item for this camp'})
+            raise ValidationError(
+                {'anchor': 'Anchor is already in use on an info item for this camp'}
+            )
+
+        if self.team.camp != self.camp:
```

Comment:
is this check needed?

Reply:
Just to make sure we don't accidentally add a 2017 team to be responsible for a 2018 info category :)

Reply:
the category FK to camp should be deleted and replaced with a camp property that returns team.camp